### PR TITLE
Add expanded rows component

### PR DESCRIPTION
**Pull request by neonwarp**

### CODE REVIEW

**Review by bartektelec — APPROVED** [review]

File: src/SvelteTable.svelte [hunk 1]
```diff
@@ -155,9 +156,11 @@
               </td>
             {/each}
           </tr>
-          {#if row['expandRow']?.show}
-            {@html row['expandRow']['component']}
-          {/if}
+          {#each columns as col}
+            {#if col.expandedRowsComponent}
```
Comment (Fedoskina) [inline src/SvelteTable.svelte:168]: Could you add a test for it?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hurtigruten/svelte-table",
-  "version": "1.0.2",
+  "version": "1.0.3",
```
Comment (Fedoskina) [inline package.json]: It looks like a new feature, probably minor update suits better

File: src/SvelteTable.svelte (hunk 1 above)
Comment (neonwarp) [reply src/SvelteTable.svelte:168]: Done

**Review by Fedoskina — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -34,4 +36,42 @@ const columns = [
...
+  {
+    key: 'Id',
+    title: 'ID',
+    value: (v) => v.Id,
+    sortable: true
+  },
+  {
+    key: 'FullName',
+    title: 'Full name',
+    component: FullName
```
Comment (bartektelec) [inline README.md:68]: Maybe add a README example with expandRowComponent? Or just list out all props

**Review by bartektelec — APPROVED** [review]

**Pull request merged**